src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -125,6 +158,16 @@ function ExamSettings(props) {
             onChange={onCreateZendeskTicketsChange}
           />
         </ValidationFormGroup>
+        { proctortrackEscalationEmailError
+        && (
+        <StatusAlert
+          alertType="danger"
+          className="proctortrackEscalationEmailError"
```

Comment:
The reason why I chose to use className is because id is not being passed through to the component (so the id never shows up in the html).

Reply:
Ah. That's too bad. Thanks for explaining. I will switch this `StatusAlert` to the Bootstrap React `Alert` component, which will be replacing the `StatusAlert` (see [here](https://edx.github.io/paragon/status/)). I bet it will allow me to set an `id`, so I can change that when I rebase on top of your work, if that works for you.